### PR TITLE
Discord Automod patch

### PR DESCRIPTION
**Pull request by User9684**
Fixes discord's stupid automod sending event several times. See [here](https://canary.discord.com/channels/166207328570441728/384011387132706816/1137801255616516096) for context

### CODE REVIEW

**Review by ashishjh-bst — APPROVED** [review]

File: automod/automod_bot.go [hunk 1]
```diff
@@ -278,6 +280,26 @@ func (p *Plugin) handleAutomodExecution(evt *eventsystem.EventData) {
...
+	var exists string
+
+	if err := common.RedisPool.Do(radix.Cmd(&exists, "GET", redisKey)); err != nil {
+		return
+	}
+	if exists == "1" {
+		return
+	}
+
+	if err := common.RedisPool.Do(radix.Cmd(nil, "SET", redisKey, "1")); err != nil {
```
Comment (ashishjh-bst) [inline automod/automod_bot.go:294]: you can use SETEX instead of doing EXPIRE in a another command. 
https://redis.io/commands/setex/

**Review by ashishjh-bst — CHANGES_REQUESTED** [review]
some minor changes are needed

File: automod/automod_bot.go (hunk 1 above)
Comment (User9684) [reply automod/automod_bot.go:294]: Ah, didn't know, I'll change that real quick.

**Comment by User9684** [comment]
Don't merge yet, sado is being annoying.

**Pull request merged**